examples/counter/store/configureStore.ts

```diff
@@ -1,13 +1,18 @@
-import {createStore, applyMiddleware, compose} from 'redux';
+import * as Redux from 'redux';
+const {createStore, applyMiddleware, compose} = Redux;
 const thunk = require('redux-thunk');
 import reducer from '../reducers/index';
-const devTools = require('redux-devTools').devTools;
+const devTools = require('redux-devtools').devTools;
```

Comment:
Might be worth promoting the chrome extension version of the redux devtools instead, since they don't require your angular app to have a dependency on react :scream:

https://github.com/zalmoxisus/redux-devtools-extension